src/ValueRepresentation.js

```diff
@@ -357,68 +365,72 @@ class BinaryRepresentation extends ValueRepresentation {
                     offsets = [];
                 }
 
+                const getFrameOrFragement = stream => {
+                    var nextTag = Tag.readTag(stream);
+                    if (!nextTag.is(0xfffee000)) {
+                        return [null, true];
+                    }
+                    let frameOrFragmentItemLength = stream.readUint32();
+                    const buffer = stream.getBuffer(
```

Comment:
Ideally rather than returning getBuffer which still slices the values (making a copy of the array), we could return a Uint8Array view with an offset and length. This seems to cause some test regression for DicomDict writing though.

This may not be such a big deal if the stream is getting GC